sphinx/users/comlinetools/conversion.rst

```diff
@@ -29,6 +29,15 @@ The output file format is determined by the extension of the output file, e.g.
 
     .. versionadded:: 5.4.0
 
+.. option:: -noflat
+
+    Do not flatten the series i.e. use the sub-resolution API for images
+    with pyramidal levels::
```

Comment:
Worth mentioning that (1) this only matters if the output format is OME-TIFF, and (2) a pyramid will not be generated without it?

This PR also resolves https://trello.com/c/whGDjGxc/316-add-noflat-to-http-docsopenmicroscopyorg-bio-formats-592-users-comlinetools-displayhtml from the backlog.

Reply:
With the new major version, wouldn't it be an opportune time to make `noflat` the default and to add `flat` options to enable flattening if desired?  This would make pyramid support the default.  We've left `flat` the default for a good six years now; isn't it time to change that?

Reply:
See f2c0cb5 for the 2 comments.

I definitely approve the idea of enabled sub-resolution by default especially now that Bio-Formats 6.0.0 has support for sub-resolution writing API and a concrete format implementing this API. I'll record this. Potentially the biggest thing to evaluate would be the impact when someone is converting to another format than `.ome.tiff` as it can lead to data loss.